src/shapes/active_selection.class.js

```diff
@@ -100,6 +100,17 @@
       return false;
     },
 
+    /**
+     * @override
+     * @return {Boolean}
+     */
+    setCoords: function () {
```

Comment:
isn't this redundant?
group has exactly this

Reply:
no it has it with subtargetcheck as an option indeed.

Reply:
I see.